public/locales/en-US/translation.json

```diff
@@ -0,0 +1,20 @@
+{
+  "common": {
```

Comment:
I wouldnt use common attributes for the translations. 

I think you will run into unwanted side effects when changing text that does not apply to all "common" occurences.